README.md

```diff
@@ -71,6 +71,19 @@ let p = new Preprocessor();
 let output = p.parse('<template>Hi</template>');
 ```
 
+#### `PreprocessorOptions`
+
+````ts
+interface PreprocessorOptions {
+
+  /** Default is `false` */
+  inline_source_map: boolean;
+
+  filename: string;
```

Comment:
Is this one optional?

Should be ?: as well?

Reply:
ahhh.... yeah you are right

Reply:
pushed a fix @NullVoxPopuli